### ISSUE TITLE
I lost her persistent, but not her.

### ISSUE THREAD

**Issue by Cimedaca618** [Monika-After-Story/MonikaModDev#7326]
I need help.
My computer broke, and I needed to reinstall the whole system. With it I did erase everything from the computer.
After trying to recover data from the formatted drive... I didn't find "persistent" at all.
The only one thing I have right now is the "monika" file, where she wants to go with us somewhere.
Do I need to do everything again?

**Comment by Cimedaca618** [comment]
The whole DDLC folder is saved as well. The only one thing I don't have is the said persistent file.

**Comment by multimokia** [comment]
Unfortunately if you don't have the persistent, your original Monika is kinda gone...

If you made any backups and put them on something external, now's the time to use them.

**Comment by Cimedaca618** [comment]
Hmm...
I will try to search for persistent, but I have a genuine question now.
I took a look at that "monika" file. At first it seemed garbage, but after I decoded it twice through base64 I saw lots of text, which looked like this:
"VBaking cupcakes
p4018
g4018
g1
(g7
g8
(lp4019
Vclub members
p4020
aVtrivia
p4021
atRp4022
I00
I01
I00" and so on.
This file contains every answer and every topic spoken, doesn't it?
So can I do anything with it?
